### ISSUE TITLE
css example not working?

### ISSUE THREAD

**Issue by voltrevo** [timreichen/Bundler#32]
Hi there

I'm having some trouble with the css functionality here and decided to try the example.

For me it doesn't seem to be working out of the box. I followed the instructions but I see that in `deps` there is a single css file that exactly mirrors `src/styles.css`:

```css
/* @import "h1.css"; */
@import "import.css";

#images {
  display: flex;
  flex-wrap: wrap;
}
```

This causes the browser to fetch `deps/import.css` which does not exist. Perhaps the css transform is just not working? Is there something I have to do to enable it?

I also really want to use the css nesting that is advertised to work out of the box but it seems something more fundamental isn't right. Can you reproduce or is it just my set-up?

Using deno 1.9.0.

**Comment by timreichen** [comment]
I will look Info that. There were changes on both esm.sh and deno 1.9 what might have broken the examples. I am working on an update which i hope to be able to release in a few days

**Comment by timreichen** [comment]
Just published 6.3. Are the problems resolved with that update?
